packages/building-rollup/README.md

```diff
@@ -1,151 +1,164 @@
+const baseConfig = createBasicConfig();
+
+export default merge(baseConfig, {
+  input: './index.html',
+});
 ```
 
-In addition to outputting a regular build of your app, it outputs a legacy build which is compatible with older browsers down to IE11.
+</details>
 
-At runtime we determine which version of your app should be loaded, so that legacy browsers don't force you to ship more and slower code to most users on modern browsers.
+#### Custom HTML template
 
-## Config features
+You can also provide the HTML as a template, as a string or read from a file. See the [html plugin docs](https://github.com/open-wc/open-wc/tree/master/packages/rollup-plugin-html#readme) for all possible options.
 
-`createDefaultConfig`:
+<details>
+<summary>View example</summary>
 
-- compatible with browsers which [support es modules](https://caniuse.com/#feat=es6-module)
-- babel transform based on browser support (no es5 for all browsers)
-- load polyfills when needed:
-  - dynamic import
-  - webcomponents
-- resolve bare imports ( `import { html } from 'lit-html'` )
-- preserve `import.meta.url` value from before bundling
-- minify + treeshake js
-- minify html and css in template literals
+```js
+import merge from 'deepmerge';
+import { createBasicConfig } from '@open-wc/building-rollup';
 
-`createCompatibilityConfig`:
+const baseConfig = createBasicConfig({
+  html: {
+    template: /* your template goes here */,
+  },
+});
 
-- Two build outputs:
+export default merge(baseConfig, {
+  input: './src/app.js',
+});
+```
+
+</details>
 
-  - Modern build:
+### Non-SPA projects
 
-    - es modules
-    - compatible with browsers which [support es modules](https://caniuse.com/#feat=es6-module)
-    - babel transform based on browser support (no es5 for all browsers)
+If you are not building a single page app you can use `createBasicConfig` to set up regular JS to JS bundling.
 
-  - Legacy build:
+From there on it's easy to extend further for different use cases. For example, to bundle and generate multiple HTML files you can take a look at the documentation of [@open-wc/rollup-plugin-html](https://github.com/open-wc/open-wc/tree/master/packages/rollup-plugin-html#readme). See below how to add plugins to the rollup config.
 
-    - systemjs modules
-    - compatible down to IE11
-    - babel transform down to IE11 (es5)
-    - core js polyfills (`Promise`, `Symbol`, `String.prototype.includes` etc.)
+## Injecting a service worker
 
-  - Both:
-    - resolve bare imports ( `import { html } from 'lit-html'` )
-    - preserve `import.meta.url` value from before bundling
-    - load polyfills when needed:
-      - dynamic import
-      - webcomponents
-      - fetch
-    - minify + treeshake js
-    - minify html and css in template literals
+When you use `createSpaConfig` a service worker is generated automatically. It is not injected into your `index.html` by default to prevent unexpected results. You can turn this on by enabling the `injectServiceWorker` option:
```

Comment:
```suggestion
When you use `createSpaConfig` a service worker is generated automatically with Workbox. The service worker registration is not injected into your `index.html` by default to prevent unexpected results. You can turn this on by enabling the `injectServiceWorker` option:
```